types/coin.go

```diff
@@ -269,6 +269,23 @@ func (coins Coins) safeAdd(coinsB Coins) Coins {
 	}
 }
 
+// ContainsDenomsOf returns true if coinsB' denom set
+// is subset of the receiver's denoms.
+func (coins Coins) ContainsDenomsOf(coinsB Coins) bool {
```

Comment:
Weren't you supposed to change this to `IsDenomSubset` @alessio?

Reply:
Forgot it, done now